diff --git a/org.inkscape.Inkscape.appdata.xml.in b/org.inkscape.Inkscape.appdata.xml.in
index 3ce960294a..b8721118e4 100644
--- a/org.inkscape.Inkscape.appdata.xml.in
+++ b/org.inkscape.Inkscape.appdata.xml.in
@@ -208,6 +208,6 @@
     <control>tablet</control>
   </supports>
   <recommends>
-    <display_length compare="ge">medium</display_length>
+    <display_length compare="ge">768</display_length>
   </recommends>
 </component>
