--- a/inkscape.appdata.xml.in	2020-04-13 16:46:13.560024032 -0400
+++ b/inkscape.appdata.xml.in	2020-04-13 16:46:42.248334473 -0400
@@ -35,12 +35,12 @@
   <releases>
     <release version="0.92.5" date="2020-04-08">
       <description>
+        <p>New 0.92.x bugfix release.</p>
         <ul>
-          <li>Bugfix release</li>
           <li>More python3 compatibility for extensions</li>
-	  <li>https://wiki.inkscape.org/wiki/index.php/Release_notes/0.92.5</li>
 	</ul>
       </description>
+      <url>https://wiki.inkscape.org/wiki/index.php/Release_notes/0.92.5</url>
     </release>
     <release version="0.92.4" date="2019-01-16">
       <description>
