diff --git a/share/CMakeLists.txt b/share/CMakeLists.txt
index 8cb35ddea2..a34bb6bdc2 100644
--- a/share/CMakeLists.txt
+++ b/share/CMakeLists.txt
@@ -15,7 +15,7 @@ add_subdirectory(templates)
 add_subdirectory(pixmaps)
 add_subdirectory(tutorials)
 add_subdirectory(ui)
-add_subdirectory(doc)
+# add_subdirectory(doc)
 
 install(DIRECTORY extensions DESTINATION ${INKSCAPE_SHARE_INSTALL}
     COMPONENT extensions
