diff --git a/org.inkscape.Inkscape.appdata.xml.in b/org.inkscape.Inkscape.appdata.xml.in
index cfd422d807..38e3d706f2 100644
--- a/org.inkscape.Inkscape.appdata.xml.in
+++ b/org.inkscape.Inkscape.appdata.xml.in
@@ -72,9 +72,9 @@
         <ul>
           <li>Bugfix release</li>
           <li>More python3 compatibility for extensions</li>
-	</ul>
-	  <url>https://wiki.inkscape.org/wiki/index.php/Release_notes/0.92.5</url>
+        </ul>
       </description>
+      <url>https://wiki.inkscape.org/wiki/index.php/Release_notes/0.92.5</url>
     </release>
     <release version="0.92.4" date="2019-01-16">
       <description>
